### PR TITLE
Update to actix-web 2.0

### PR DESCRIPTION
**Pull request by DominicWrege**
Content changed  to actix-web 2.0:
- index
- getting started
- examples/main-example

### CODE REVIEW

**Comment by JohnTitor** [comment]
Thanks! But I'll let #132 go first if other members like it.

**Comment by DominicWrege** [comment]
It's fine. I only updated one example ```main-example```. I am ok if we use your version. 
Primarly I updated the website.

**Comment by JohnTitor** [comment]
Okay, #132 has been merged. Could you resolve conflicts and rebase?

**Comment by DominicWrege** [comment]
Done

File: layouts/shortcodes/actix-rt-version.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{{- if eq (.Get 0) "actix-rt" -}}
+    {{- .Page.Site.Params.actixRtVersion -}}
+{{- end -}}
```
Comment (JohnTitor) [inline layouts/shortcodes/actix-rt-version.html]: This can be placed on actix-version.html, not separated files.

**Comment by JohnTitor** [comment]
I commented one minor nit, otherwise the change looks good to me.

File: layouts/shortcodes/actix-rt-version.html (hunk 1 above)
Comment (DominicWrege) [reply layouts/shortcodes/actix-rt-version.html]: No Problem.

File: layouts/shortcodes/actix-version.html [hunk 2]
```diff
@@ -3,3 +3,7 @@
 {{- else if eq (.Get 0) "actix-web" -}}
   {{- .Page.Site.Params.actixWebVersion -}}
 {{- end -}}
+
+{{- if eq (.Get 0) "actix-rt" -}}
+    {{- .Page.Site.Params.actixRtVersion -}}
+{{- end -}}
```
Comment (JohnTitor) [inline layouts/shortcodes/actix-version.html]: Use else if:

```
{{- if eq (.Get 0) "actix" -}}
  {{- .Page.Site.Params.actixVersion -}}
{{- else if eq (.Get 0) "actix-web" -}}
  {{- .Page.Site.Params.actixWebVersion -}}
{{- else if eq (.Get 0) "actix-rt" -}}
  {{- .Page.Site.Params.actixRtVersion -}}
{{- end -}}
```

Comment (DominicWrege) [reply layouts/shortcodes/actix-version.html]: got it

**Comment by DominicWrege** [comment]
Done

**Pull request merged**